### PR TITLE
octopus: qa/distros: move to latest version on supported distro's

### PR DESCRIPTION
**Pull request by ideepika**
what changes?

all distro references updates:
rhel to rhel_8 ( rhel 8.3 )
centos to centos_8 ( centos 8.2 )
ubuntu_latest -> ubuntu 20.04 (adds rook pkg failures to ignorelist)
ubuntu 18.04 only tests non-cephadm suites

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by tchaikov** [comment]
@ideepika thank you Deepika. could you note in the commit message explaining why this commit is not cherry-picked from master?

**Comment by ideepika** [comment]
> @ideepika thank you Deepika. could you note in the commit message explaining why this commit is not cherry-picked from master?



> @ideepika thank you Deepika. could you note in the commit message explaining why this commit is not cherry-picked from master?

@tchaikov thought earlier they were divergent, but tried to find and cherry pick ones I could, and fixed some octopus only stuff, see: https://github.com/ceph/ceph/pull/41478#issue-649930491

File: qa/suites/rados/cephadm/dashboard/0-distro/.qa [hunk 1]
```diff
@@ -0,0 +1 @@
+../.qa
```
Comment (tchaikov) [inline qa/suites/rados/cephadm/dashboard/0-distro/.qa:1]: this change actually **adds** a new test suite to `qa/suites/rados/cephadm/dashboard`. is this intended?

**Comment by tchaikov** [comment]
> > @ideepika thank you Deepika. could you note in the commit message explaining why this commit is not cherry-picked from master?
>
> @tchaikov thought earlier they were divergent, but tried to find and cherry pick ones I could, and fixed some octopus only stuff, see: [#41478 (comment)](https://github.com/ceph/ceph/pull/41478#issue-649930491)

this comment is still not fully addressed.

**Comment by tchaikov** [comment]
> summary of changes:
> [ in supported_random_distro, supported_distros and latest distro files added in distro/all along with symlink updates ]
> 
> changes:
> [1]
> rhel_8.yaml -> 8.3
> centos_8.yaml -> 8.3
> ubuntu_latest -> 20.04 (also would solve issues with cephadm, as podman
> deprecated support for bionic)
> 
> [2] not sure, from what I remember `$` helps with creating job config, thought might be need, please let me know if otherwise.

please read https://github.com/ceph/teuthology/commit/e77d436db95345b19b93e59dc6d68745d6c03f2b

File: qa/suites/rados/cephadm/dashboard/0-distro/.qa (hunk 1 above)
Comment (ideepika) [reply qa/suites/rados/cephadm/dashboard/0-distro/.qa]: @tchaikov I addressed most of the issues with PR and also removed some of the additional suites that were introduced with cherrypicks, added them in trivial fix comment in commits. Can you help review, thanks!

**Comment by sebastian-philipp** [comment]
the QA run for this PR failed. I think #41568 causes the issue, but I'm not 100% sure. 
his is my analysis for the run: https://tracker.ceph.com/issues/51272

**Comment by ideepika** [comment]
failure unrelated, is because my clone didn't had nautilus fork, rest all green :-) 
http://pulpito.front.sepia.ceph.com/ideepika-2021-06-23_09:55:26-rados-wip-deepika2-testing-2021-06-23-0828-octopus-distro-basic-smithi/
http://pulpito.front.sepia.ceph.com/ideepika-2021-06-23_20:56:30-rados-wip-deepika2-testing-2021-06-23-0828-octopus-distro-basic-smithi/ 

codependent PR's existing with wip-deepika2-testing tag

**Review by idryomov — COMMENTED** [review]
I wonder if "qa/distro/supported-random-distro: add $ file" is actually necessary because there already is an `$` in the path.  If everything worked before without it, why is it needed now?

**Comment by ideepika** [comment]
> I wonder if "qa/distro/supported-random-distro: add $ file" is actually necessary because there already is an `$` in the path. If everything worked before without it, why is it needed now?

makes sense, my reasoning was just to keep things more and more in sync with upstream/pacific, can remove :)

**Review by yuriw — APPROVED** [review]

**Comment by ideepika** [comment]
https://trello.com/c/7j47hYIy/1247-wip-yuri8-testing-2021-06-15-0839-octopus-old-wip-yuri8-testing-2021-06-02-1014-octopus-old-wip-yuri8-testing-2021-05-24-0950-oc

**Pull request merged**